main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -444,6 +464,26 @@ object ParsedAst {
       */
     case class QName(sp1: SourcePosition, name: Name.QName, sp2: SourcePosition) extends ParsedAst.Expression
 
+    /**
+      * Simply Qualified Name Expression: The qualifier is either or a class or a top-level namespace.
+      *
+      * @param sp1 the position of the first character in the expression.
+      * @param qualifier the namespace or class.
+      * @param name the name.
+      * @param sp2 the position of the last character in the expression.
+      */
+    case class SQName(sp1: SourcePosition, qualifier: Name.Ident, name: Name.Ident, sp2: SourcePosition) extends ParsedAst.Expression
```

Comment:
qualifier is the class name and name is the sig name?

Reply:
Yes, but also no.

`SQName` is ambiguous: it could be `Class.sig`, but it could also be `Namespace.def`. That's why the names are a bit vague.